src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { ChangeEventHandler, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [reseach, setReseach] = useState('');
+  const searchedMovie = (movies: Movie[], searchedText: string) => {
+    const searchedInput = searchedText.toLocaleLowerCase().trim();
+
+    return movies.filter(movie => {
+      const titleAndDescription = `${movie.title.toLocaleLowerCase().trim()}${movie.description.toLocaleLowerCase().trim()}`;
```

Comment:
Interesting approach here,  I would probably prefer searching separately at title and description, but why not.